bootstrap/README.md

```diff
@@ -1,42 +1,49 @@
 # Bootstrapping cabal-install
 
-This utility is only intended for use in building cabal-install
-on a new platform. If you already have a functional (if dated) cabal-install
-please rather run `cabal v2-install`.
+This utility is only intended for use in building `cabal-install`
+on a new platform. If you already have a functional cabal-install, however old, then
+please instead run `cabal install cabal-install`.
 
-The typical usage is porting to a new linux architecture,
-then the `linux-{ghc-ver}.json` file is available in the `bootstrap/` folder:
+## Bootstrapping on Linux
 
-On a (linux) system you are bootstrapping, run
+The typical use case is porting to a new Linux architecture. In that case,
+a `linux-${GHCVER}.json` file is available in the `bootstrap/` folder:
 
-   ./bootstrap/bootstrap.py -d ./bootstrap/linux-ghcver.json -w /path/to-ghc
+On a Linux system you are bootstrapping, run
+
+    ./bootstrap/bootstrap.py -d ./bootstrap/linux-${GHCVER}.json -w /path/to-ghc
 
 from the top directory of the source checkout.
 
+### Offline build
+
 For offline builds, you can first run
 
-   ./bootstrap/bootstrap.py -d ./bootstrap/linux-ghcver.json -w /path/to-ghc fetch
+    ./bootstrap/bootstrap.py -d ./bootstrap/linux-${GHCVER}.json -w /path/to-ghc fetch
 
 to fetch tarballs for all the dependencies. These can then be used by a further
 bootstrap command by way of the `--bootstrap-sources` argument:
 
-   ./bootstrap/bootstrap.py -w /path/to-ghc --bootstrap-sources bootstrap-sources.tar.gz
+    ./bootstrap/bootstrap.py -w /path/to-ghc --bootstrap-sources bootstrap-sources.tar.gz
 
-To generate the `platform-{ghc-ver}` files for other platforms, do:
+## Bootstrapping on other (non-Linux) platforms
```

Comment:
maybe other (non-Linux) unix platforms

Reply:
Again, a bit judgemental. I don't feel like `unix platforms` mean anything today. So I'd rather avoid this vague term.